### PR TITLE
TASK: document @ignoreProperties in TYPO3.TypoScript:Array

### PR DESCRIPTION
**Pull request by dimaip**

### CODE REVIEW

**Comment by kitsunet** [comment]
👍 probably needs to be documented in other places too, but every step helps :)

**Comment by dimaip** [comment]
@kdambekalns fixed. All fine now?

File: TYPO3.Neos/Documentation/References/NeosTypoScriptReference.rst [hunk 1]
```diff
@@ -17,6 +17,7 @@ TYPO3.TypoScript:Array
 Render multiple nested definitions and concatenate the results.
 
 :[key]: (string) A nested definition (simple value, expression or object) that evaluates to a string
+:[key].@ignoreProperties: (array) A list of properties to ignore from being "rendered" during evaluation
```
Comment (hlubek) [inline TYPO3.Neos/Documentation/References/NeosTypoScriptReference.rst]: Small nitpick: Why quotes here?

Wouldn't "properties to ignore from being evaluated during rendering" be correct?

**Comment by hlubek** [comment]
Anyway, let's merge this small change

**Pull request merged**